### ISSUE TITLE
Update Vet Center designs to include Events & side nav 

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#17194]
## User Story or Problem Statement

Vet Center pages don't currently have side navigation menus, because they are essentially single pages (except for one secret sub page). As we add Events, we will need to give these pages side nav menus. 

Consider the existing VAMC designs and whether there are any opportunities to update them

- Events listing page: https://www.va.gov/chicago-health-care/events/
- Specific events page: https://www.va.gov/outreach-and-events/events/62368/ 
- Address any accessibility concerns found in #16346

Dave will draft the side nav contents in #17195 

## Acceptance Criteria
- [ ] Designs are documented in Figma
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review
- [ ] Requires CAIA IA review (likely directly with Mikki, sync with Amanda on this)
- [ ] Requires accessibility review

**Comment by aklausmeier** [comment]
may need to split VBA and Vet Centers into dedicated tickets as they will both need CAIA's review

**Comment by jilladams** [comment]
We're considering pulling this design ticket into Sprint 1, but the side nav IA ticket isn't complete. @davidmpickett @thejordanwood shoudl this ticket properly be blocked by #17195?

**Comment by davidmpickett** [comment]
> We're considering pulling this design ticket into Sprint 1, but the side nav IA ticket isn't complete. @davidmpickett @thejordanwood shoudl this ticket properly be blocked by #17195? (Assuming so for now, and added the dependency.)

I would not consider it blocking. 95% of Jordan's work will unimpacted by the contents of the side navs. The only real pertinent design implication is how many levels deep the nesting will go. I don't think we need more than two levels.

![Screenshot 2024-03-27 at 9 58 16 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/feab8f04-897d-4f40-acfc-dc30a96e856f)

vs the three levels on VAMC

![Screenshot 2024-03-27 at 9 58 29 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/64a4da22-281a-4a5e-850a-480f095f98e6)

We've also talked about using Figma designs as the Content Source of Truth so we're not managing two different documents. I can draft the side nav directly in Figma if this ticket is done first.

The follow up step that these two tickets are collectively blocking is likely something like **prepping for Design Intent for Vet Center/VBA Events**

**Comment by davidmpickett** [comment]
Split VBA events into separate ticket: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18139

**Comment by davidmpickett** [comment]
Closing as a duplicate of $15349 which has a lot more context
